examples/manipulating_visualizing/plot_neurovault_meta_analysis.py

```diff
@@ -0,0 +1,99 @@
+"""
+NeuroVault meta-analysis of stop-go paradigm studies.
+=====================================================================
+
+This example shows how to download statistical maps from
+NeuroVault
+
+See :func:`nilearn.datasets.fetch_neurovault` documentation for more details.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+from nilearn import datasets
+
+### Define filters ############################################################
+cfilts = []  # [lambda col: col['DOI'] is not None]
+imfilts = [lambda im: "stop signal" in (im.get('cognitive_paradigm_cogatlas') or "").lower(),
```

Comment:
Just for some quick feedback: this is the JSON dict from neurovault. It can return blank or `None`, or sometimes the value isn't present at all. The idiom above is how I typically handle such cases.

To simplify the example, I could try to normalize things inside the `fetch` function. Because I'm not sure what to expect from Neurovault (no documentation to that level) and after talking with @chrisfilo about the differences in metadata provided from different data sources, returning the raw JSON from `fetch` and letting users handle what they see, seemed like the safest thing.